src/Features/Core/Portable/AddPackage/AbstractAddPackageCodeFixProvider.cs

```diff
@@ -58,7 +58,7 @@ internal abstract partial class AbstractAddPackageCodeFixProvider : CodeFixProvi
                 searchNugetPackages &&
                 installerService.IsEnabled(document.Project.Id))
             {
-                foreach (var packageSource in installerService.GetPackageSources())
+                foreach (var packageSource in await installerService.TryGetPackageSourcesAsync(allowSwitchToMainThread: false, cancellationToken).ConfigureAwait(false))
```

Comment:
Isn't this missing the null check? It seems the compiler lets you enumerate a Nullable<ImmutableArray<...>> and implicitly does the .Value for you, but that'll throw at runtime if it's null.